src/plugins/genericPlugin/plugin.ts

```diff
@@ -43,7 +44,7 @@ export class GenericPlugin extends ProposalPlugin<
   IGenericPluginState,
   IGenericPluginProposalState,
   IProposalCreateOptionsGS
-> {
+  > {
```

Comment:
why additiional space is here ?

Reply:
This was generated by my editor. I'm not sure what rules it is following, but the change makes sense by aligning the bracket with the lines above.  Looks clean and harmless enough to me.